### PR TITLE
Improve/rename auditor

### PR DESCRIPTION
**Pull request by eric-intuitem**
auditor -> reader

### CODE REVIEW

**Comment by ab-smith** [comment]
impact on existing instance?

**Comment by eric-intuitem** [comment]
No impact expected, as the code of the group has not been changed.

File: frontend/messages/fr.json [hunk 1]
```diff
@@ -367,7 +367,7 @@
 	"extraLarge": "Très grand",
 	"rid": "RID",
 	"scope": "Périmètre",
-	"auditor": "Auditeur",
+	"reader": "Auditeur",
```
Comment (nas-tabchiche) [inline frontend/messages/fr.json]: The translation in french stays the same @eric-intuitem?

Comment (eric-intuitem) [reply frontend/messages/fr.json]: Indeed. Solved.

**Review by Mohamed-Hacene — APPROVED** [review]
0k

**Pull request merged**